example/particles/particles.cpp

```diff
@@ -218,22 +230,28 @@ TaskStatus CreateSomeParticles(MeshBlock *pmb) {
           vy(n) = v * sin(theta) * sin(phi);
           vz(n) = v * cos(theta);
 
+          // Create particles at the beginning of the timestep
+          t(n) = t0;
+
           weight(n) = 1.0;
 
           rng_pool.free_state(rng_gen);
         }
       });
 
+  swarm->swarm_num_incomplete_ = swarm->get_num_active();
+
   return TaskStatus::complete;
 }
 
-TaskStatus TransportParticles(MeshBlock *pmb, StagedIntegrator *integrator) {
+TaskStatus TransportParticles(MeshBlock *pmb, StagedIntegrator *integrator, double t0) {
```

Comment:
Do you account for particle particle interactions somewhere?

Reply:
This example doesn't include any notion of particle-particle interactions, depending on how you wanted to implement that your `Transport` task might need to look different.